Makefile

```diff
@@ -9,22 +9,23 @@ PWD = $(shell pwd)
 TOX = tox
 
 $(VENV):
-	virtualenv .venv
-	$(VENV_ACTIVATE); pip install tox bumpversion twine
+	virtualenv $(VENV)
+	$(VENV_ACTIVATE); pip install tox bumpversion twine 'readme_renderer[md]'
 	$(VENV_ACTIVATE); pip install -e .
 
 lint: $(VENV)
-	tox -e flake8
+	$(VENV_ACTIVATE); tox -e flake8
 
-test: clean $(VENV) lint
-	cd $(PWD)/tests/test-two-envs && $(TOX)
-	cd $(PWD)/tests/test-env-inheritance && $(TOX)
-	cd $(PWD)/tests/test-environment-variable && ./run-tox.sh $(TOX)
+test: clean-tests $(VENV) lint
+	$(VENV_ACTIVATE); cd $(PWD)/tests/test-two-envs && $(TOX)
+	$(VENV_ACTIVATE); cd $(PWD)/tests/test-env-inheritance && $(TOX)
+	$(VENV_ACTIVATE); cd $(PWD)/tests/test-environment-variable && ./run-tox.sh $(TOX)
 
-dist: clean-dist
+dist: clean-dist $(VENV)
 	python setup.py sdist
```

Comment:
Why upload sdist rather than wheels? It's pure python, so I think we should avoid having to make users run setup.py.

Reply:
No reason really. It can be switched to wheels if we want.